src/disclosures/js/index.js

```diff
@@ -3,10 +3,13 @@
 var fetch = require( './dispatchers/get-api-values' );
 var financialModel = require( './models/financial-model' );
 var schoolModel = require( './models/school-model' );
+var getModelValues = require( './dispatchers/get-model-values' );
+var getUrlValues = require( './dispatchers/get-url-values' );
```

Comment:
Yep, I realized I forgot it in the shower this morning, and I was hoping to sneak it in. Coming soon!